configuration/gunicorn.conf.py

```diff
@@ -0,0 +1,9 @@
+bind = '0.0.0.0:8000'
+workers = 2
+worker_connections = 1000
+threads = 2
+timeout = 30
+keepalive = 32
+max_requests = 100
+user = 'nobody'
+accesslog = '-'
```

Comment:
FYI, the default gunicorn setup uses
```python
import multiprocessing as mp
workers = mp.cpu_count() * 2 + 1
```
But it's NOT always true. It's depends on your server. Sometimes it's too much when operations on your server isn't cpu bound. Sometimes it's not enough.

**P.S.** You don't need to change that 'cause your laptop could fucked up.